spectrum/src/main/java/com/thebluealliance/spectrum/internal/ColorCircleDrawable.java

```diff
@@ -13,6 +13,7 @@
     private final Paint mPaint;
     private int mRadius = 0;
     private int mOutlineWidth = 0;
+    private @ColorInt int mOutlineColor = -1;
```

Comment:
Whoops, forgot to remove that, that was before I realized there was already a setOutlineColor method in that class